frontend/ui/elements/screen_rotation_menu_table.lua

```diff
@@ -24,9 +25,29 @@ return {
             })
         end
 
+        if Device:hasGSensor() then
+            table.insert(rotation_table, {
+                text = _("Lock auto rotation to current orientation"),
+                help_text = _([[When checked, the gyro will only be honored when switching between the two inverse variants of your current rotation,
+i.e., Portrait <-> Inverted Portrait OR Landscape <-> Inverted Landscape.
+Switching between (Inverted) Portrait and (Inverted) Landscape will be inhibited.
```

Comment:
You can put the first line at the start of a new line (so the whole text is left aligned and the code is easier to read).
Lua strips out the firt `\n`.